browsers.js

```diff
@@ -103,12 +103,12 @@ export async function createBrowserWorker( url, browser, options, restarts = 0 )
 	worker.browser = browser;
 	worker.restarts = restarts;
 	worker.options = options;
-	touchBrowser( browser );
```

Comment:
This touch was never actually applied. The following line added the worker to the list, which is needed in `touchBrowser`. It isn't necessary anyway and was immediately deleted.